lnbits/app.py

```diff
@@ -454,6 +459,16 @@ def log_server_info():
     logger.info(f"Service fee wallet: {settings.lnbits_service_fee_wallet}")
 
 
+async def init_latest_payment_hash():
```

Comment:
why not running the crud function directly where it is needed, i dont see the point of prefetching it into settings?
you could utilize the cache utility for this instead of settings

Reply:
- I didn't want to introduce a dependency from `wallet` to `crud`
- it is a minor/fast operation